### ISSUE TITLE
nmap shows space instead of <space>

### ISSUE THREAD

**Issue by chelovechishko** [vifm/vifm#385]
`:nmap` shows ' ' instead of `<space>`, the same for `<tab>`.

**Comment by xaizek** [comment]
Yes, that's because `<*>` are replaced on mapping and `:nmap` doesn't know which spaces are from `<space>` and which aren't. To fix this original user input needs to be saved when mapping is set, which isn't done at the moment.
